sql/pgwire/v3.go

```diff
@@ -622,8 +622,8 @@ func (c *v3Conn) sendResponse(resp sql.Response, formatCodes []formatCode, sendD
 		return c.sendCommandComplete(nil)
 	}
 	for _, result := range resp.Results {
-		if result.PErr != nil {
-			if err := c.sendError(result.PErr.String()); err != nil {
+		if result.Err != nil {
+			if err := c.sendError(result.Err.Error()); err != nil {
```

Comment:
I don't know that we're trying to send a `roachpb.Error` back. I thought it would be a structured error (after all, we only expose one type of retry condition to the SQL client) which belongs to SQL. I think it's OK to drain roachpb.Error from the callpath for the time being where possible - I don't think pgwire should have to know about a KV specifics.